### PR TITLE
Added Current booted sha to comment in template header metadata

### PR DESCRIPTION
**Pull request by ninjapanzer**
# Description

Added current sha of GSD-tools to the header metadata of gsd-web

This will not throw off the current UI layout as it adds no visible content.

This value is extracted when quasar is booted from the underlying .git folder

Does not add any additional dependencies.

The sha is written into index.template.html so the ssr artifact can be free of the git metatdata when used.

Fixes https://github.com/cloudsecurityalliance/gsd-tools/issues/50

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

- [x] Booted the application and verified that the build version is in the footer

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: gsd-web/src/boot/buildVersion.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+// Uses code from https://stackoverflow.com/a/34518749
+
+const fs = require('fs');
+
+const rev = fs.readFileSync('../.git/HEAD').toString().trim();
```
Comment (athix) [inline gsd-web/src/boot/buildVersion.js:5]: I'm guessing what happens here is that we end up baking this into the dist files upon compilation, meaning that we don't have to worry about production not necessarily having the git meta alongside the dist.

File: gsd-web/quasar.conf.js [hunk 2]
```diff
@@ -9,6 +9,7 @@
 /* eslint-env node */
 const ESLintPlugin = require('eslint-webpack-plugin')
 const { configure } = require('quasar/wrappers');
+const buildSha = require('./src/boot/buildVersion');
```
Comment (athix) [inline gsd-web/quasar.conf.js:12]: Similar to #51, we aren't injecting this into the boot sequence, so I suspect this file should live elsewhere.

File: gsd-web/src/boot/buildVersion.js (hunk 1 above)
Comment (ninjapanzer) [reply gsd-web/src/boot/buildVersion.js:5]: I am seeing that this fails on build for some reason. So I am going to close this for now and find a new solution,

**Comment by ninjapanzer** [comment]
Closing because this will not complete a build.

**Comment by athix** [comment]
One note for if you find an alternative solution @ninjapanzer: Putting this as a meta tag in the header might be more appropriate than an HTML comment in the body. That way it's guaranteed not to affect the DOM, and should be easier to find and even automatically parse if needed.

**Comment by ninjapanzer** [comment]
@athix Sounds good, figured out the issue and moved this into a meta header.

I can confirm that in this case where I am adding it to the root template the git sha is compiled into the ssr artifact.

**Review by athix — APPROVED** [review]
Tested locally and confirmed working as intended, both for dev and prod. (compiles meta header into html template)

**Pull request merged**